docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+
+Also add a literal block named `rounds`. This block contains each benchmark test round that is to be run headed by a unique round label. Rounds may be used to benchmark different smart contract methods, or the same method in a different manner. 
+
+The `round` block contains the following: 
+
+- `label` - the unique header label to use for the round.
+- `description` - a description of the round being run.
+- `txDuration` - the specification of the test duration, in seconds
+- `rateControl` - a rate control type, with options. 
+- `workload` - the workload module to use, with arguments to pass to the module. All arguments passed are available as `roundArguments` within the workload module.
+
+This is how it should be populated: 
+
+``` bash
+rounds:
+    - label: queryAsset
+      description: Query asset benchmark
+      txDuration: 30
+      rateControl: 
+        type: fixed-load
+        opts:
+          transactionLoad: 2
+      workload:
+        module: benchmarks/workload/queryAssetBenchmarks.js
+        arguments:
+          assets: 10
+          contractID: fabcar
+```
+
+### Resulting File
+
+The benchmark configuration file should now be fully populated. 
+
+It should look like this:
+
+```bash
+test:
+    name: my-asset-benchmark
+    description: test benchmark
+    workers:
+      type: local
+      number: 2
+    rounds:
+      - label: queryAsset
+        description: Query asset benchmark
+        txDuration: 30
+        rateControl: 
+          type: fixed-load
+          opts:
+            transactionLoad: 2
+        workload:
+          module: benchmarks/workload/queryAssetBenchmarks.js
+          arguments:
+            assets: 10
+            contractID: fabcar
+```
+
+## Step 6 - Running and Result
+
+### Setting up the command
+
+We are now ready to run the performance benchmark using the above materials we have created. The performance benchmark will be using the Caliper CLI and will also need to paths to the network configuration file, the benchmark configuration file, and the workspace. 
+
+That should be as follows: 
+
+- caliper-networkconfig: `networks/network_config.json`
+- caliper-benchconfig: `benchmarks/myAssetBenchmark.yaml`
+- caliper-workspace: `./`
+
+Since the chaincode has already been installed and instantiated the only actions caliper needs to perform is the test phase using a fabric gateway that has discovery enabled. To specify this you should add the following additional flags to the CLI command:
+
+- `caliper-flow-only-test`
+- `caliper-fabric-gateway-enabled`
+- `caliper-fabric-gateway-discovery`
+
+### Run the command
+
+Ensure that you are in the caliper-workspace directory. 
+
+In the terminal run the following Caliper CLI command:
+
+`caliper launch manager --caliper-benchconfig benchmarks/myAssetBenchmark.yaml --caliper-networkconfig networks/network_config.json --caliper-workspace ./ --caliper-flow-only-test --caliper-fabric-gateway-enabled --caliper-fabric-gateway-discovery`
```

Comment:
This command failed for 2 reasons
1. long couldn't be found
2. The connection profile from fabric now confirms to a new standard and the configValidator fails. Long story but now tlsCACerts supports arrays as well as strings and the CA definition from test-network uses that format

Reply:
removed the ca aspect from the tutorial as the v2 connector when using a gateway wont need it